src/core/src/UsdUtil.cpp

```diff
@@ -367,20 +367,31 @@ pxr::CesiumData getOrCreateCesiumData() {
     return defineCesiumData(CesiumDataPath);
 }
 
-pxr::SdfPath getPathToCurrentIonServer() {
+pxr::SdfPath getPathToOfficialIonServer() {
     return pxr::SdfPath("/CesiumServers/IonOfficial");
 }
```

Comment:
This function doesn't seem to be used, and I'm not sure I like that the path is hardcoded. See if it can be removed.